### DISCUSSION TITLE
grpc_status label missing for response_latency histogram

### DISCUSSION THREAD

**Discussion by bjor-joh** [linkerd/linkerd2#9709] [Help]
Is the response_latency_ms_bucket proxy metric suppose to have grpc_status as a label? Or is that only present on response_total ?
The [documentation](https://linkerd.io/2.12/reference/proxy-metrics/#response-labels) makes it sound like they should be there
But I can’t actually see it, only the http status code is there.

It's a bit problematic because then I can filter on response time for successful requests separately. Is it a bug or is the documentation not up to date?

**Comment by apoorva11029** [comment]
@bjor-joh we are running into a similar issue for another metric. Curious if you found the resolution? Thanks!
